packages/core/src/auth/tokens/disposable-token-scope.ts

```diff
@@ -77,6 +77,10 @@ export type DisposableTokenScope =
   | PredefinedScope
   | DisposableTokenCachePermissions;
 
+export interface DisposableTokenProps {
+  tokenID?: string;
```

Comment:
here - I think we want `tokenId` for this public interface unless you have some style reference that suggests otherwise